packages/build/tests/core/tests.js

```diff
@@ -281,6 +281,48 @@ test('--apiHost is used to set Netlify API host', async (t) => {
   t.snapshot(requests)
 })
 
+test('Print warning when redirects file is missing from publish directory', async (t) => {
+  await runFixture(t, 'missing_redirects_warning', {
+    flags: { featureFlags: { netlify_build_warning_missing_headers: true } },
+  })
+})
+
+test('Does not print warning for missing redirects file without the feature flag', async (t) => {
+  await runFixture(t, 'missing_redirects_warning', {
+    flags: { featureFlags: { netlify_build_warning_missing_headers: false } },
+  })
+})
+
+test('Does not print warning when redirects file is not missing from publish directory', async (t) => {
+  await runFixture(t, 'missing_redirects_present', {
+    flags: { featureFlags: { netlify_build_warning_missing_headers: true } },
+  })
+})
+
+test('Does not print warning when redirects file is missing both from the build directory', async (t) => {
```

Comment:
```suggestion
test('Does not print warning when redirects file is missing from the build directory', async (t) => {
```

Reply:
If I understood correctly in this scenario the redirects file is within the `publish` directory but not the `build` one right?

Reply:
Yes! Fixed.